### DISCUSSION TITLE
Operation did not complete successfully because the file contains a virus or is potentially unwanted software

### DISCUSSION THREAD

**Discussion by JFuqX** [dashingsoft/pyarmor#1020] [Q&A] [answered]
This is new since the 8.x.x update and I just tried to revert but still got the same after running the compiled one file.

``DLL load failed while importing pytransform_vax: Operation did not complete successfully because the file contains a virus or is potentially unwanted software``

**Comment by jondy** [comment]
pytransform_vax has JIT code, please configure your system to allow JIT in pytransfrom_vax.so

**Comment by jondy** [comment, marked as answer]
someone else says
> I tried the command on a different folder and the application was not flagged but when I use it on a certain folder, it always gets flagged and I do not know why.

What I thought of 
* First mark `pytransform3.pyd` and `pyarmor_runtime_vax_xxxxx.pyd` is safe in any path in windows defender
* Check documentation of Windows Defender
* Ask question in MSDN

Also refer to
https://answers.microsoft.com/en-us/windows/forum/all/operation-did-not-complete-successfully-because/8b071316-3a5a-4ae9-973b-cfb2f36ae783

Note that windows defender may be upgraded by Microsoft automatically.
